src/App.js

```diff
@@ -1,38 +1,87 @@
 import React, { Component } from 'react';
 import './App.css';
 
+const BlockType = {
+  Heading: 'heading',
+  Text: 'text',
+  Image: 'image',
+  Blockquote: 'blockquote',
+};
+
 class App extends Component {
   state = {
     blocks: [
-      { type: 'heading', text: 'hello world' },
-      { type: 'text', text: 'another string' },
-      { type: 'image', url: 'http://fpoimg.com/300x300', text: 'FPO image' },
-      { type: 'blockQuote', text: 'this is a quote' },
-    ].map(this.withKeys),
+      { type: BlockType.Heading, text: 'hello world' },
+      { type: BlockType.Text, text: 'another string' },
+      {
+        type: BlockType.Image,
+        url: 'http://fpoimg.com/300x300',
+        text: 'FPO image',
+      },
+      { type: BlockType.Blockquote, text: 'this is a quote' },
+    ].map(this.toObjectsWithKeys),
   };
 
-  withKeys(block, index) {
+  toObjectsWithKeys(block, index) {
```

Comment:
Sorry for doing this to you, but I actually like `toObjectWithKey` (singular) better now that I see it.